main.go

```diff
@@ -25,15 +28,41 @@ import (
 )
 
 func main() {
-	err := builder.APIServer.
-		// +kubebuilder:scaffold:resource-register
-		WithResourceAndHandler(&appcatv1.AppCat{}, appcat.New()).
-		WithResourceAndHandler(&appcatv1.VSHNPostgresBackup{}, postgres.New()).
-		WithoutEtcd().
+
+	var appcatEnabled bool = false
+	var vshnEnabled bool = false
+
+	if os.Getenv("APPCAT_HANDLER_ENABLED") == "1" {
+		appcatEnabled = true
+	}
+	if os.Getenv("VSHN_POSTGRES_BACKUP_HANDLER_ENABLED") == "1" {
+		vshnEnabled = true
+	}
```

Comment:
```suggestion
    vshnEnabled, err = strconv.ParseBool(os.Getenv("VSHN_POSTGRES_BACKUP_HANDLER_ENABLED"))
	if err != nil {
	    klog.Printf("Failed to parse VSHN_POSTGRES_BACKUP_HANDLER_ENABLED: %s", err.Error())
		vshnEnabled = false
	}
```

I'd suggest using `ParseBool` to set the environment variable. That way `true` or `True` or `TRUE` or `T` all still work.